### PR TITLE
Fixes copy paste to work with the marker

### PR DESCRIPTION
**Pull request by alschmiedt**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves
Copy paste was broken for keyboard navigation.

### Proposed Changes
If we are in accessibility mode and a connection has been marked then try to insert the copied block at the marked location. If the marker is not at a valid connection point, then warn the user. 

### Test Coverage

Tested on:

### CODE REVIEW

File: core/keyboard_nav/navigation.js [hunk 1]
```diff
@@ -542,6 +542,9 @@ Blockly.Navigation.insertBlock = function(block, targetConnection) {
   if (bestConnection && bestConnection.isConnected() &&
       !bestConnection.targetBlock().isShadow()) {
     bestConnection.disconnect();
+  } else if (!bestConnection) {
+    Blockly.Navigation.warn(
+        'This block can not be inserted at the marked location. Moved to workspace.');
```
Comment (RoboErikG) [inline core/keyboard_nav/navigation.js]: Does this move the target connection to the workspace in this case or leave it where it is?

Comment (alschmiedt) [reply core/keyboard_nav/navigation.js]: It leaves the marked connection where it is.

Comment (RoboErikG) [reply core/keyboard_nav/navigation.js]: Update the warning to say it's being left where it is then.

Comment (alschmiedt) [reply core/keyboard_nav/navigation.js]: The block is moved to the workspace which is what I was trying to communicate with the "Moved to workspace". Would it make more sense for it to be "Block has been moved to the top level workspace?"

Comment (RoboErikG) [reply core/keyboard_nav/navigation.js]: In the copy paste case it is, but this method also gets called if you select a block in a stack and press 'i' to insert it. In that case, if the marked location is invalid does it stay in the stack or get moved to the workspace?

Comment (alschmiedt) [reply core/keyboard_nav/navigation.js]: ah yup forgot about that. I will take that part out since it doesn't apply to all scenarios.

**Pull request merged**